### DISCUSSION TITLE
Falang and Extra article options

### DISCUSSION THREAD

**Discussion by sbouey** [templaza/astroid-framework#270] [General]
Hi,

There are problem beetween Falang and Astroid framework on the extra tab on the article 
the article type option can't be load without breaking the options tab.
there are no problem with the Blog Options and Open Graph tab.

i have to desactivate the file libraries/astroid/framework/forms/article.xml

i'm using the dev version of Falang with change to start to fix this tab problem i can send you the modified file 

Stéphane
Falang team

**Comment by Chacapamac** [comment]
@sbouey 
**Do you have a screenshot or if you website is online?**

**I just check the last Astroid Framewor/Template version and Joomla 4 on PHP 8.1**
No problems...
— I also select all type options without problems.

![Screen Shot 2023-01-07 at 2 04 51 PM](https://user-images.githubusercontent.com/1907276/211166602-acde1634-731f-4661-a4c4-373540801063.jpg)

**Comment by sbouey** [comment]
Hi here is the screen capture of article translation with or without the article.xml of the astroid framework.
it's a test site not online, i have a Falang user with problem on astroid , i don't understand why this problem occur only with article.xml

![astroid_with](https://user-images.githubusercontent.com/11226212/211268641-a967676c-e187-412c-b794-2ab373ea8295.png)
![astroid_without](https://user-images.githubusercontent.com/11226212/211268952-0ad6d534-7dc8-4f15-96f4-69580a99ec82.png)

**Comment by Chacapamac** [comment]
Ok, I think I understand, tell me if I’m wrong

• In **Falang**, in the **translation panel** the normal Joomla “**Options**” pane disappear.
• Removing the “**article.xml**” make the **Astroid “Article Type”** pane disappear but the Joomla “**Option**” pane show in the Falang Translation.


**_Under I just put the normal Joomla Article options menu (with Astroid) for reference_**

![Screen Shot 2023-01-09 at 3 51 12 PM](https://user-images.githubusercontent.com/1907276/211407375-41f4077f-ef01-44dd-94b0-bc06b2ca8bde.jpg)

**Comment by sbouey** [comment]
yes exactly , it's happen only for the the "Article Type Options" ,  i don't know if it's a Falang problem or the way the article.xml is loaded.
Stéphane

**Comment by Chacapamac** [comment]
**Can you give an access to the website to take a look?**
Send info to —> p.toulze@grafcomm.ca

**Comment by sbouey** [comment]
Hi,

I haven't see this reply 2 years ago....but I will release version 5.14 which fixes this bug
The problem was in Falang , the form was loaded twice to have the original and translated parameter but this mean the function 

`   public function onContentPrepareForm($form, &$data)
`

with 
`  Form::addFormPath(JPATH_SITE . '/' . $astroid_dir . '/framework/forms');`

The second time the path was not right and the default attribs was removed.

Stéphane
